cluster-scope/base/core/namespaces/nexodus/kustomization.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```

Comment:
Just assuming that this file should be moved from apex-qa to nexodus-qa?

Reply:
Git does some weird things when it comes to renaming which files to what when changes like this happen. All 3 namespaces exist, even if its changing the old prod one to current qa